contract/src/error.rs

```diff
@@ -129,8 +129,8 @@ pub enum NFTCoreError {
     MissingMigrationFlag = 124,
     InvalidMigrationFlag = 125,
     ContractAlreadyMigrated = 126,
-    UnregisteredOwnerInMint = 127,
-    UnregisteredOwnerInTransfer = 128,
+    UnregisteredOwnerInPageAdd = 127,
```

Comment:
Lets keep the original names of the errors, although its an error code, I'd like to keep the names consistent from minor release

Reply:
@darthsiroftardis I get the consistency matter but the fact is that `UnregisteredOwnerInMint` could then be triggered into a `Transfer` and that seems to me confusing for the user.

Shall I add two new errors, and `UnregisteredOwnerInMint` / `UnregisteredOwnerInTransfer` get deprecated ?

Reply:
@darthsiroftardis As suggested let me know what you think of this revert and flag thanks!

https://github.com/casper-ecosystem/cep-78-enhanced-nft/pull/144/commits/d9456b97be9fe07d7c4e6aeb3d1d50745ff8e524

Reply:
made a simple flag true/false